src/components/EmptyState.tsx

```diff
@@ -1,15 +1,17 @@
-import React, { FC } from "react";
+import React, { FC, ReactNode } from "react";
 import { Button, Col, Icon, Row } from "@canonical/react-components";
 
 interface Props {
   iconName: string;
   iconClass: string;
   title: string;
-  message: string;
-  linkMessage: string;
-  linkURL: string;
-  buttonLabel: string;
-  buttonAction: () => void;
+  message: string | ReactNode;
+  linkMessage?: string;
+  linkURL?: string;
+  buttonLabel?: string;
+  buttonAction?: () => void;
+  isDisabled?: boolean;
+  customButton?: ReactNode;
```

Comment:
The interface of EmptyState is getting a bit chaotic. How about we just output children in it? Children can be one or multiple buttons. Then we don't need the button* isDisabled, customButton, etc props here.

@yurii-vasyliev build a nice and clean component for [anbox](https://github.com/canonical/anbox-cloud-dashboard/blob/main/static/js/components/layout/EmptyState.tsx) that might serve as inspiration how to keep the interface tidy.